### PR TITLE
Support monicoins in compacting drawers in hard/expert.

### PR DESCRIPTION
**Pull request by tomprince**
Also, remove the unused monipenny in those modes.

### CODE REVIEW

File: kubejs/startup_scripts/registry/item_registry.js [hunk 1]
```diff
@@ -458,3 +460,12 @@ StartupEvents.registry('item', event => {
...
 })
+
+ItemEvents.toolTierRegistry(event => {
+    event.add("dev", (tier) => {
+        tier.enchantmentValue = 42
+        tier.level = 42
+        tier.uses = 42690
+        tier.speed = 12
+    })
+})
```
Comment (Niki4tap) [inline kubejs/startup_scripts/registry/item_registry.js]: Unrelated change?

File: kubejs/server_scripts/monicoins/monicoins.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
-ServerEvents.recipes(event => {
```
Comment (Niki4tap) [inline kubejs/server_scripts/monicoins/monicoins.js:1]: Pretty sure this line shouldn't be removed

Comment (OrioPrisco) [reply kubejs/server_scripts/monicoins/monicoins.js:1]: i think so too, but if you really wanted to remove it properly you'd need to remove the matching parenthesis at the end of the file

File: kubejs/startup_scripts/registry/item_registry.js (hunk 1 above)
Comment (tomprince) [reply kubejs/startup_scripts/registry/item_registry.js]: It looks like both of these are artifacts of a bad rebase. Fixed.

Comment (Niki4tap) [reply kubejs/startup_scripts/registry/item_registry.js]: Ah, happens, it looks like the other comment is still valid though...

**Comment by ThePansmith** [comment]
All good, but removing monipenny is kinda random, keep it

**Comment by tomprince** [comment]
> All good, but removing monipenny is kinda random, keep it

I wanted to *not* add monipenny -> moninickel compacting recipe in hard/expert, to avoid getting a penny-nickel-quarter drawer if you put the wrong item in a drawer first. There is no way (other than downcrafting) to get any pennies, since they are not given as quest rewards. To be consistent with the compacting drawers, I thought it makes sense to remove the coin, given that there are no uses in hard/expert. 

 since none of the quests give you them, and the only way to get them would be to manually craft them (for no useful purpose).

**Comment by ThePansmith** [comment]
actually yeah fair enough
